packages/ember-lifeline/src/utils/timer.ts

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+import { TestTimeoutOptions } from '../types';
+
+/**
+ * @function getTimeoutOrTestFallback
+ * @param {number} timeoutVal The timeout value that gets scaled down in case of the test environment.
+ * @param {number} [customTestTimeout] The timeout value that needs to be set in the test environment if passed
+ */
+export function getTimeoutOrTestFallback(
+  timeoutVal: number,
+  options?: TestTimeoutOptions
+): number {
+  if (Ember.testing) {
+    const customTestTimeout = options?.timeout;
```

Comment:
Can we just call this `timeout`?

```suggestion
    const timeout = options?.timeout;
```

Reply:
Since there is another param named `timeout`, I couldn't rename it to timeout. Instead, i have renamed it to `testTimeout`.